doc/source/cluster/index.rst

```diff
@@ -4,42 +4,69 @@
 
 .. _cluster-index:
 
-Ray Cluster Overview
-====================
+Ray Clusters Overview
+=====================
 
 What is a Ray cluster?
-------------------------
+----------------------
 
-One of Ray's strengths is the ability to leverage multiple machines in the same
-program. Ray can, of course, be run on a single machine (and is done so often),
-but the real power is using Ray on a cluster of machines.
+One of Ray's strengths is the ability to leverage multiple machines for distributed execution. Ray can, of course, be run on a single machine (and is done so often), but the real power is using Ray on a cluster of machines.
 
-A Ray cluster consists of a **head node** and a set of **worker nodes**. The
-head node needs to be started first, and the worker nodes are given the address
-of the head node to form the cluster:
+Ray can automatically interact with the cloud provider to request or release instances. You can specify :ref:`a configuration <cluster-config>` to launch clusters on :ref:`AWS, GCP, Azure, Kubernetes, Aliyun, on-premise, and Staroid or even on your custom node provider <cluster-cloud>`. Your cluster can have a fixed size or :ref:`automatically scale up and down<cluster-autoscaler>` depending on the demands of your application.
```

Comment:
Staroid is no longer available -- I don't remember if we've removed references to it elsewhere in the docs, but we should.